### PR TITLE
Fix setting High file priority on qBittorrent 3.2.x+

### PR DESCRIPTION
**Pull request by bwitt**
Store the qBittorrent priority values in the adapter class and then adjust when the api version is known.

Fixes https://github.com/erickok/transdroid/issues/619

### CODE REVIEW

**Comment by bwitt** [comment]
I tested this with qBittorrent v4.3.9

File: app/src/main/java/org/transdroid/daemon/adapters/qBittorrent/QBittorrentAdapter.java [hunk 1]
```diff
@@ -124,10 +128,16 @@ private synchronized void ensureVersion(Log log) {
             }
 
             version = parseVersionNumber(versionText);
+            if (version >= 30200) {
+              qbNormalPriority = 6;
```
Comment (erickok) [inline app/src/main/java/org/transdroid/daemon/adapters/qBittorrent/QBittorrentAdapter.java:132]: This is the same for versions >= 30200 and 40100. Is that what you intented?

**Review by erickok — APPROVED** [review]
Thanks for your contribution! I will merge, just have 1 question.

File: app/src/main/java/org/transdroid/daemon/adapters/qBittorrent/QBittorrentAdapter.java (hunk 1 above)
Comment (bwitt) [reply app/src/main/java/org/transdroid/daemon/adapters/qBittorrent/QBittorrentAdapter.java:132]: yes they should be the same; now I realize we probably don't need or want to be explicit about what happens at each version (also got the ordering wrong) so I'm going to remove the block for ` version >= 40100`

**Review by erickok — APPROVED** [review]

**Pull request merged**